docker-compose.yml

```diff
@@ -27,19 +27,21 @@ services:
     ports:
       - "8090:8090"
     healthcheck:
-      test: "python3 /example-app/ostis-web-platform/sc-machine/scripts/healthcheck.py"
+      test: "/bin/bash -c 'source /example-app/.venv/bin/activate && python3 /example-app/scripts/healthcheck.py'"
       interval: 5s
       timeout: 10s
       retries: 3
-      start_period: 120s
+      start_period: 0s
     environment:
       # Use the commented env variable if you need to rebuild KB every startup.
       #- "REBUILD_KB=1"
```

Comment:
I enabled rebuild and docker compose up failed
![image](https://github.com/user-attachments/assets/83244f00-2dbd-41a6-8796-79deaa70cc89)

Reply:
Fixed